src/components/Preview.js

```diff
@@ -105,29 +136,80 @@ class Preview extends Component {
           <h2>Preview</h2>
           <InfoIcon infoText={previewText} />
         </div>
-        <div className={classNames({ 'Preview-muted': !numFrames })}>
+        <div className="Frame-section-container">
+          <img
+            className="directional-icon"
+            src={left}
+            onClick={() =>
+              this.handleChangePreviewIdx(Math.max(previewIdx - 1, 0))
+            }
+            alt=""
+          />
           <Frame
             imageSrc={imageSrc}
             playing={playing}
             togglePlaying={this.handleTogglePlaying}
           />
-          <div className="Preview-scrubber" data-testid="Preview-scrubber">
-            <input
-              type="range"
-              min="0"
-              max={numFrames - 1}
-              value={previewIdx}
-              onChange={this.handlePreviewUpdate}
-              disabled={!numFrames}
-              aria-label="preview frame index"
+          <img
+            className="directional-icon"
+            src={right}
+            onClick={() =>
+              this.handleChangePreviewIdx(
+                Math.min(previewIdx + 1, frameIDs.length - 1)
+              )
+            }
+            alt=""
+          />
+        </div>
+        <div className="Frame-delete">
+          {!!numFrames ? (
+            <button
+              className="Frame-delete-redo-button"
+              aria-label="delete this frame"
+              onClick={() => this.handleDeleteFrame(previewIdx)}
+            >
+              Delete this Frame
+            </button>
+          ) : null}
+          {!!redoFrames.length ? (
+            <button
+              className="Frame-delete-redo-button"
+              aria-label="redo last frame"
+              onClick={this.handleRedoFrame}
+            >
+              Redo Last Delete
```

Comment:
I'm confused: a user deletes a frame and then is posed with the ability to 'Redo Last Delete'? What does that mean?

Reply:
The wording is a bit ambiguous, changed to `Undo Delete` for clarity. Thoughts?

Reply:
Love it.